test/integration/multiplexed_upstream_integration_test.cc

```diff
@@ -573,6 +573,33 @@ TEST_P(MultiplexedUpstreamIntegrationTest, LargeResponseHeadersRejected) {
   EXPECT_EQ("503", response->headers().getStatusValue());
 }
 
+// Tests configuration of max response headers size.
+TEST_P(MultiplexedUpstreamIntegrationTest, LargeResponseHeadersAccepted) {
+  constexpr uint32_t limit_kb = 150;
```

Comment:
can we test the actual bounds (within 1k of max) just to make sure all 3 codecs allow headers that large?

Reply:
Good call. It turns out there are a couple different limits I ran into, and the request header size knobs weren't testing the limit. I documented the limits in the protobuf.